packages/components/src/components/action-pad/action-pad.browser.e2e.tsx

```diff
@@ -98,4 +103,100 @@ describe("calcite-action-pad", () => {
       "calcite-action-group",
     );
   });
+
+  describe("keyboard navigation and selection-mode", () => {
+    it("supports ARIA keyboard navigation and focus management", async () => {
```

Comment:
Can you update this test to also include selection via keyboard (i.e., `Enter`)?